utils/updateTestPlanReadme.pl

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env perl
+
+use strict;
+use warnings;
+
+use Pod::Usage;
+use Getopt::Long;
+use File::Basename;
+
+# command line options
+my $dirloc = '.'; # directory for serching test cases
+my $help;
+my $DEBUG = 0;
+
+# global variables
+my $tcCounter = 0;
+
+sub main {
+    GetOptions(
+        'help|?'  => \$help,
+        'dir|d=s' => \$dirloc,
+        'debug'   => \$DEBUG,
+    ) or pod2usage(2);
+
+    if ($help) {
+        pod2usage(1);
+        exit;
+    }
+    opendir(my $dir, $dirloc) || die "cannot open directory $dirloc";
+
+    my @files = readdir $dir;
+    @files = sort {$a cmp $b} @files;
+    foreach my $f (@files) {
+        print "$dirloc/$f\n" if $DEBUG;
```

Comment:
Maybe output diagnostics to STDERR? Not that it makes a huge difference in the short term, but it is a good practice.

Reply:
Done.